### ISSUE TITLE
peer dependency question

### ISSUE THREAD

**Issue by wcaneira** [testing-library/eslint-plugin-testing-library#142]
Thank you for building this plugin, it's useful.

Just a question, please: I noticed that it (indirectly, thru it's dependencies) requires a peer dependency of `typescript` which I'm not using in my project.  

```console
└─┬ eslint-plugin-testing-library@3.1.3
  └─┬ @typescript-eslint/experimental-utils@2.34.0
    └─┬ @typescript-eslint/typescript-estree@2.34.0
      └── tsutils@3.17.1 
```

```console
npm WARN tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
```

...any way around this warning, or am i stuck with it?

**Comment by Belco90** [comment]
Hello! I can't see that warning in my projects without TS using this plugin. That's a util we are using since we moved the plugin to TS. I'm not sure why you are seeing that warning and I'm not. I understand what peer dependencies are but I don't get the difference to see the warning in some envs or not.

Anyway, I think there is no problem getting that kind of warning? Probably you'll have to live with it. I can see tons of similar warnings from other ESLint stuff in my local env.

<img width="794" alt="image" src="https://user-images.githubusercontent.com/2677072/82645065-97e60500-9c12-11ea-8c76-945996c153ae.png">


I'll keep an eye on this and let you know if I can get more info around this. Interesting how npm will change this behavior in v7 tho, I guess all this peer dependency stuff is not really clear for most people.

**Comment by wcaneira** [comment]
thank you for replying.  you are right, the warning may not be a big deal functionally, it's just that in my experience, the more warnings accumulate over time, people tend to ignore them all as noise - and some warnings should be heeded. :-)

so im trying to limit/eliminate warnings when folks run `npm i` in my app, so that only true problems appear.

thank you again for replying.

**Comment by Belco90** [comment]
You are welcome! I would like to have a better response actually, and I understand your concerns around several warnings related to peer dependencies. As I said, I'll let you know if I can find more details about it.
